src/arch/riscv/machine/hardware.c

```diff
@@ -102,6 +102,10 @@ static inline irq_t getActiveIRQ(void)
          * has claimed it in a multicore system.
          */
         irq = plic_get_claim();
+#ifdef CONFIG_PLAT_QEMU_RISCV_VIRT
+         // unmatched writes to claim are ignored by hardware
```

Comment:
Can you extend the comment to mention the counterpart in `invokeIRQHandler_AckIRQ()`, like the comment there mentions this. Then we have the connection, in case QEMU's behavior changes in future version.

Reply:
Done.